strings/names/religion_meme.txt

```diff
@@ -0,0 +1,16 @@
+Homosexuality
+lol
+wtf
+gay
+penis
+ass
+poo
+badmin
+shitmin
+deadmin
+cock
+cocks
+meme
+memes
+weeaboo
+kawaii
```

Comment:
Let's not.

Reply:
I am but the messenger

Reply:
Remove this crap. There's enough creativity in the playerbase without the game needing to add this kind of suggestions.